Nodejs cpp (#226)
aspose-pdf/Aspose.PDF-Documentation: nodejs-cpp/get-started/extract-text/_index.md

---
title: Extract Text from PDF using Node.js
linktitle: Extract Text from PDF
type: docs
weight: 10
url: /nodejs-cpp/extract-text/
description: This section describes how to extract text from PDF document using Node.js toolkit.
lastmod: "2023-11-14"
sitemap:
    changefreq: "weekly"
    priority: 0.7
---

## Extract Text From all the Pages of PDF Document

> Extracting text from PDF isn’t easy. Not many PDF readers can extract text from PDF images or scanned PDFs. But **Aspose.PDF for Node.js via C++** tool allows you to easily extract text from all PDF file in the Node.js environment. 

Reviewer: ```suggestion
Extracting text from PDF isn’t easy. Only a few PDF readers can extract text from PDF images or scanned PDFs. But **Aspose.PDF for Node.js via C++** tool allows you to easily extract text from all PDF file in the Node.js environment. 
```
